### PR TITLE
Theta2 fixes

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: fact_plots/scripts/plot_theta_squared.py [hunk 1]
```diff
@@ -117,22 +117,26 @@ def main(data_path, threshold, theta2_cut, key, bins, alpha, start, end, prelimi
 
     del events
 
-    limits = [0, 0.3]
```
Comment (jebuss) [inline fact_plots/scripts/plot_theta_squared.py:120]: Is max_theta2 = 0.3 a result of trigonometry, 5 + 1 wobble positions and a wobble offset of 0.6 degree? 
I mean, does 0.3 reflect the radius of a circle around a wobble position, so no circles overlap?

Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: No, this is just what turned out to look nice.

Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: Wobble distance of 0.6 degrees would mean that the maximum theta2 without overlap would be 0.36 deg²

Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: Ah, no. wrong. It's smaller.

Comment (jebuss) [reply fact_plots/scripts/plot_theta_squared.py:120]: If this is the case? Can the limit be calculated automatically from the wobble offset and the number of source positions?
If not could you ad a comment telling what this number means

**Review by jebuss — CHANGES_REQUESTED** [review]

File: fact_plots/scripts/plot_theta_squared.py (hunk 1 above)
Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: Why? What has the maximum theta we show in a theta square plot something to do with that?

Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: FYI, the maximum theta² cut you can do without overlapping regions should be

(wobble_distance * sin(π / (n_off + 1)))²


Which is 0.089 for 5 off positions and 0.6° wobble distance

Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: But again, this has imho nothing to do with the xrange of the theta2 plot.

Comment (jebuss) [reply fact_plots/scripts/plot_theta_squared.py:120]: Ok, thanks for the clarification. I was just stumbling over this number and was wondering if it has a deeper meaning ;-). If it does not, leaving a comment what come handy to not misunderstand it, but is not mandatory imho.

**Review by jebuss — APPROVED** [review]

File: fact_plots/scripts/plot_theta_squared.py (hunk 1 above)
Comment (maxnoe) [reply fact_plots/scripts/plot_theta_squared.py:120]: I missed a 0, which is pretty significant in this case ;-)

Comment (jebuss) [reply fact_plots/scripts/plot_theta_squared.py:120]: I was mislead because (wobble_distance * sin(π / (n_off + 1))) = 0.3, with n_off=5

**Pull request merged**